src/tests/mocks/erc721_receiver_mocks.cairo

```diff
@@ -0,0 +1,218 @@
+use openzeppelin::tests::utils::constants::SUCCESS;
+
+#[starknet::contract]
+mod DualCaseERC721ReceiverMock {
+    use openzeppelin::introspection::src5::SRC5Component;
+    use openzeppelin::token::erc721::ERC721ReceiverComponent;
+    use starknet::ContractAddress;
+
+    component!(path: ERC721ReceiverComponent, storage: erc721_receiver, event: ERC721ReceiverEvent);
+    component!(path: SRC5Component, storage: src5, event: SRC5Event);
+
+    // ERC721Receiver
+    impl ERC721ReceiverImpl = ERC721ReceiverComponent::ERC721ReceiverImpl<ContractState>;
+    impl ERC721ReceiverInternalImpl = ERC721ReceiverComponent::InternalImpl<ContractState>;
+
+    // SRC5
```

Comment:
I like this sections approach. If we are separating like this with section comments, maybe we should do the same with the other mocks? (in ERC721 for ex.). No strong need of course, but pushing for consistency